ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/BonsaiPersistedWorldState.java

```diff
@@ -82,7 +82,10 @@ public MutableWorldState copy() {
             worldStateStorage.codeStorage,
             worldStateStorage.storageStorage,
             worldStateStorage.trieBranchStorage,
-            worldStateStorage.trieLogStorage));
+            worldStateStorage.trieLogStorage,
+            Pair.with(
+                worldStateStorage.snapTrieBranchBucketsStorage.getValue0(),
+                worldStateStorage.snapTrieBranchBucketsStorage.getValue1())));
```

Comment:
Pair and getValue0 and getValue1 doesn't communicate our purpose in using a tuple.  If we use an explicit class that encapsulates the two buckets/transactions that would help clarify what is going on - we could have class level javadoc that indicates what is going on with snap and the 2 bucket approach.